actions/feedback_actions.py

```diff
@@ -38,6 +38,9 @@
         FuzzySlotMatchOption(
             "bug",
             [
+                "Yes, it's a bug",
+                "yes",
+                "yes a bug",
```

Comment:
I think it's better if we detect `bug` in the  sentence or the `intent_core_yes` , but i guess this works for now.

Reply:
That makes sense, I'll add it back :)